pkg/policies/opa/rego/gcp/google_container_node_pool/cosNodeImageUsed.rego

```diff
@@ -2,6 +2,7 @@ package accurics
 
 cosNodeImageUsed[api.id]{
     api := input.google_container_node_pool[_]
-    node := api.config.node_config[_] 
-    node.image_type != "cos"
-}
\ No newline at end of file
+    node := api.config.node_config[_]
+    not startswith(node.image_type, "cos")
```

Comment:
I would suggest compare if `lower(node.image_type)` doesn’t start with “cos” to prevent false positives when setting the image type in uppercase

Reply:
Good point. Thanks!